tests/base/src/test/java/org/wildfly/security/util/PasswordBasedEncryptionUtilTest.java

```diff
@@ -183,9 +183,9 @@ private void checkPb(String secret, String salt, int iteration, String pbGenerat
         String crossDecrypted = new String(decryptUtil.decodeAndDecrypt(pbGenerated));
         String decrypted = new String(decryptUtil.decodeAndDecrypt(encrypted));
 
-        Assert.assertTrue("Elytron in PB compatible mode failed", decrypted.equals(secret));
-        Assert.assertTrue("PicketBox encrypted, Elytron decrypted in compatible mode, failed", crossDecrypted.equals(secret));
-        Assert.assertTrue("Elytron in compatible mode encrypted, PicketBox encrypted must be the same", pbGenerated.equals(encrypted));
+        Assert.assertEquals("Elytron in PB compatible mode failed", decrypted, secret);
```

Comment:
Thank you @ericrleung for the PR! 
Very minor, but could you please change the order of the values so it is following the format assertTrue( message, expectedValue, actualValue) ? Thank you! Here, I believe `secret` and `encrypted` are the expected values.